docs/upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -24,6 +24,40 @@ There you can find links to upgrade notes for other versions too.
         + image: adminer:4.7
         ```
     - run `docker-compose up -d` so the new image is pulled and used
+- production environment disallow administrators to log in with default credentials [#1360](https://github.com/shopsys/shopsys/pull/1360)
+    - register `user_checker` in  `app/config/packages/security.yml`
+        ```diff
+        administration:
+            pattern: ^/(admin/|efconnect|elfinder)
+        +   user_checker: Shopsys\FrameworkBundle\Model\Security\AdministratorChecker
+            anonymous: ~
+            provider: administrators
+            logout_on_user_change: true
+        ```
+        - in case you need to disable this functionality (e.g. to prevent failing tests on CI which uses production environment)
```

Comment:
I'm missing info about native installation. eg. that you need to add new environment variable and it should be OK

Reply:
What do you meaan by the `native instalation`? What you describe seems to mi like the second part of sentence right below.